server/views/pages/create/additionalConditions.njk

```diff
@@ -0,0 +1,123 @@
+                            conditional: {
+                                html: govukInput({
+                                    name: "supplementary-data[residence-condition-id-0][probation-region]",
+                                    classes: "govuk-!-width-one-third",
+                                    label: {
+                                        text: "Enter probation region"
+                                    }
+                                })
+                            }
+                        }
+                    ]
+                }) }}
+
+                {% set restrictionOfResidencyHtml %}
+                    {{ govukRadios({
+                        name: "supplementary-data[restriction-of-residency-condition-id-0][gender]",
+                        fieldset: {
+                            legend: {
+                                text: "Select the relevent text"
+                            }
+                        },
+                        items: [
+                            {
+                                value: "any",
+                                text: "any"
+                            },
+                            {
+                                value: "any female",
+                                text: "any female"
+                            },
+                            {
+                                value: "any male",
+                                text: "any male"
+                            }
+                        ]
+                    }) }}
+
+                    {{ govukRadios({
+                        classes: "govuk-radios--inline",
+                        name: "supplementary-data[restriction-of-residency-condition-id-0][age]",
+                        fieldset: {
+                            legend: {
+                                text: "Select the relevent age"
+                            }
+                        },
+                        items: [
+                            {
+                                value: "16",
+                                text: "16 years"
+                            },
+                            {
+                                value: "18",
+                                text: "18 years"
+                            }
+                        ]
+                    }) }}
+                {% endset %}
+
+                {{ govukCheckboxes({
+                    name: "additional-conditions",
+                    fieldset: {
+                        legend: {
+                            text: "Restriction of residency",
+                            isPageHeading: true,
+                            classes: "govuk-fieldset__legend--l"
+                        },
+                        classes: "govuk-!-margin-top-6"
+                    },
+                    items: [
```

Comment:
This is good for now (having the list of options in the template). Longer term, I wonder about pulling these options into a JSON config file and supplying the options objects into the templates. We could also version these - i.e. increment a licence version number whenever we change one of the configuration items. Discuss!

Reply:
I had a think about this, and I only added in a subset of the options into html template for now since we will eventually be pulling them out into some kind of field config. Not sure how we want this to work, since each checkbox can have different kinds of input nested inside of them. I came across this which allows html-as-config, but not sure this is the route you want to go down https://json2html.com/